src/domains/community/detail/page/UserPosts.jsx

```
@@ -1,8 +1,9 @@
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
```

ESLint

[error] 'useEffect' is defined but never used.

(no-unused-vars)